### PR TITLE
edit and delete user counts

### PR DESCRIPTION
**Pull request by EverardoAlcarazBaltazar**
# Description

I implement a modal with a form to update or delete accounts because it is necessary to allow users to update or delete their accounts.

## Screenshots

Screenshot from 2023-04-12 16-28-16

### CODE REVIEW

**Review by oscaregr — CHANGES_REQUESTED** [review]
do a git rebase before to merge

**Review by AangelF — APPROVED** [review]

**Review by oscaregr — APPROVED** [review]

**Review by DiegoMoralesBravo — APPROVED** [review]

File: app/assets/stylesheets/pages/_edit.scss [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    text-decoration: none;
+    transition: background-color 0.15s ease-in-out;
+  }
+
+  .btn-danger:hover {
+    background-color: $cancel-hover;
+    color: $color-white;
+  }
+}
+}
```
Comment (YilKanda) [inline app/assets/stylesheets/pages/_edit.scss:140]: Add a blank line.

File: app/assets/stylesheets/utilities/_variables.scss [hunk 2]
```diff
@@ -36,5 +36,17 @@ $color-white: #FFFF;
...
+$message-color: #333333;
+$message-border: 1px solid #DDD;
+$close-color: #aaa;
+$close-hover: #ff0000;
+$field-border: #cccccc;
+$field-color: #999;
+$update-color: #4CAF50;
+$update-hover: #3E8E41;
+$cancel-color: #dc3545;
+$cancel-hover: #B71C1C;
```
Comment (YilKanda) [inline app/assets/stylesheets/utilities/_variables.scss:52]: Add a blank line

Comment (YilKanda) [inline app/assets/stylesheets/utilities/_variables.scss:52]: Sort the variables

File: app/views/devise/registrations/_edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    pattern: "[a-z0-9A-Z ÁáÉéÍíÓóÚú,.]+", required: true %>
+  </div>
+  <div class= buttonsContainer>
+    <div class="actions">
+       <%= f.submit "Update", onclick: "addH2()" %>
+    </div>
+<% end %>
+      <%= button_to "Delete Count", registration_path(@resource_name), class: "btn-danger", method: :delete, onclick: "return confirm('Are you sure you want to delete your account?');" %>
+    </div>
+    <%= javascript_include_tag 'modalAlert' %>
```
Comment (YilKanda) [inline app/views/devise/registrations/_edit.html.erb:45]: Add a blank line

**Review by YilKanda — APPROVED** [review]

**Pull request merged**